switchcraft/data/computercraft/lua/rom/programs/http/pastebin.lua

```diff
@@ -78,7 +80,7 @@ local function printTryAgain(errResponse)
   end
 end
 
-local function get(url)
+local function get(url,pw)
```

Comment:
I would add a space between here, considering the code style other places in the file.
```suggestion
local function get(url, pw)
```